packages/scene-composer/src/utils/entityModelUtils/lightComponent.ts

```diff
@@ -76,8 +76,42 @@ export const createLightEntityComponent = (light: ILightComponent): ComponentReq
   return comp;
 };
 
-export const updateLightEntityComponent = (light: ILightComponent): ComponentUpdateRequest => {
+export const updateLightEntityComponent = (
+  light: ILightComponent,
+  oldComponent?: ILightComponent,
+): ComponentUpdateRequest => {
   const request = createLightEntityComponent(light);
+  if (oldComponent) {
```

Comment:
You don't need this `if (oldComponent)` check as the following `if` statement covers this.